hudi/tests/docker/Dockerfile

```diff
@@ -18,7 +18,8 @@ ENV PATH /usr/local/sbt/bin:${PATH}
 RUN wget -P /opt https://dlcdn.apache.org/maven/maven-3/3.6.3/binaries/apache-maven-3.6.3-bin.tar.gz \
     && tar -xzf /opt/apache-maven-3.6.3-bin.tar.gz 
 
-RUN apk add git && git clone https://github.com/apache/hudi.git && cd hudi/ && /apache-maven-3.6.3/bin/mvn --projects packaging/hudi-spark-bundle --also-make clean package -DskipTests -Dspark3 -Dscala-2.12 
+RUN apk add git && git clone --depth 1 --branch release-0.10.0-rc2 https://github.com/apache/hudi.git
```

Comment:
Should we pass the release version through tox env? So we easily know which version of Hudi we're testing